packages/algob/sample-project/test/sample-test.js

```diff
@@ -0,0 +1,71 @@
+import { Runtime, StoreAccount, types } from '@algorand-builder/runtime';
+
+import { getProgram } from '../../build';
+const { assert } = require('chai');
+
+const minBalance = 1000000n;
+const masterBalance = 10000000n;
+const amount = 1000000n;
+
+describe('Sample Test', function () {
+  let master = new StoreAccount(masterBalance);
+  let fundReceiver = new StoreAccount(minBalance);
+
+  let runtime;
+  const approvalProgram = getProgram('fee-check.teal');
```

Comment:
```suggestion
  const feeCheckProgram = getProgram('fee-check.teal');
```
approvalProgram represents a stateful smart contract - [https://developer.algorand.org/docs/features/asc1/stateful/#the-lifecycle-of-a-stateful-smart-contract](https://developer.algorand.org/docs/features/asc1/stateful/#the-lifecycle-of-a-stateful-smart-contract)